src/app/item-popup/ItemDetails.tsx

```diff
@@ -71,7 +71,7 @@ function ItemDetails({ item, extraInfo = {}, defs }: Props) {
 
       <ItemExpiration item={item} />
 
-      {!item.stats && item.collectibleHash !== undefined && isD2Manifest(defs) && (
+      {!item.stats && item.collectibleHash && isD2Manifest(defs) && (
```

Comment:
This is one of those cases where using falsy leads to a "0" getting rendered, I think. At least that was the worry

Reply:
true wow it's been a while since i saw a lookup among the JSX and not before return, thanks